go.mod

```diff
@@ -36,9 +36,9 @@ require (
 	github.com/go-logr/logr v1.3.0
 	github.com/gogo/protobuf v1.3.2
 	github.com/golang/protobuf v1.5.3
-	github.com/google/cel-go v0.16.1
+	github.com/google/cel-go v0.18.2
```

Comment:
I think this doesn't work with current k8s deps

Reply:
OK, will revert this depend to see if CI can succeed

Reply:
k8s depend on github.com/google/cel-go v0.17.7